core/src/main/scala/org/http4s/blaze/channel/nio1/NIO1SocketServerGroup.scala

```diff
@@ -125,7 +125,7 @@ class NIO1SocketServerGroup(pool: SelectorLoopPool) extends ServerChannelGroup {
 
               // check to see if we want to keep this connection
               if (acceptConnection(address)) {
-                clientChannel.setOption[java.lang.Boolean](java.net.StandardSocketOptions.TCP_NODELAY, false)
+                clientChannel.setOption[java.lang.Boolean](java.net.StandardSocketOptions.TCP_NODELAY, true)
```

Comment:
I think `true` here is the best default since Blaze does try to saturate the TCP/IP buffer if there is enough data. If not, then then there is no point for the network layer to wait for it either.